_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Self-host maps server
+nav_order: 6
+---
+
+# Self-host maps server
+
+The self-host maps server for OpenSearch Dashboards allows users to access the default maps service on air-gapped environments. The server provides a map manifest url with map tiles and vectors, a map tiles url, and a map vectors url that are compatible with OpenSearch.
+
+The following steps walk through setting up and using the self-host maps server with OpenSearch Dashboards.
+
+You can access the `maps-server` image via the OpenSearch official [Docker Hub repository](https://hub.docker.com/u/opensearchproject).
+{: .note}
+
+## Pull Docker image
+
+Open Docker and then open your terminal to run the following command:
```

Comment:
The "open docker" step only applies to people using Docker Desktop. You could remove "Open Docker" or you could replace it with "After installing Docker, make sure that the Docker daemon is running. Then, open your terminal [...]"